### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by karthikbhargavan**
Adding a few templates for issues. Please look and discuss. I could not find a template for security issues

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+1. Version of F*
+2. Version of KaraMeL
```
Comment (msprotz) [inline .github/ISSUE_TEMPLATE/bug-report.md:15]: do we expect bug reports to involve the verification toolchain? or should we ask for which source code version they used (either hacl-packages tagged release, or git revision from this repo)?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+3. Run steps
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (msprotz) [inline .github/ISSUE_TEMPLATE/bug-report.md:22]: not sure screenshots are relevant

**Review by msprotz — APPROVED** [review]
Looks ok but please fix comments before merging, thanks!

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (R1kM) [reply .github/ISSUE_TEMPLATE/bug-report.md:15]: Could we specify how to get both versions? (fstar.exe --version and krml -version)?

**Review by R1kM — APPROVED** [review]

**Review by polubelova — APPROVED** [review]

**Pull request merged**